api/tests/UserTest.php

```diff
@@ -11,6 +11,7 @@ class UserTest extends TestCase
 
     public function testCreateUserDefaultRoles()
     {
+        $this->markTestSkipped();
```

Comment:
Maybe we could leave comments about why these are skipped? I am curious myself, but as it stands need to find someone to ask :)

Reply:
```suggestion
        $this->markTestSkipped('Todo: Fix. https://github.com/GCTC-NTGC/gc-digital-talent/issues/1480');
```